File: TRNMNT.Core/Model/Round/MatchDetailsModel.cs
namespace TRNMNT.Core.Model.Round
{
    public class MatchDetailsModel
    {
        public string MatchId { get; set; }
        public int AParticipantPenalties { get; set; }
        public int BParticipantPenalties { get; set; }
        public int AParticipantAdvantages { get; set; }
        public int BParticipantAdvantages { get; set; }
        public int AParticipantPoints { get; set; }
        public int BParticipantPoints { get; set; }

        public int Countdown { get; set; }
        public bool IsStarted { get; set; }
        public bool IsPaused { get; set; }
        public bool IsCompleted { get; set; }
    }
}

File: TRNMNT/ClientApp/app/core/model/match-details.model.ts
import { MatchModel } from './match.models';

export class MatchDetailsModel {
    matchId: string;
    aParticipantPenalties: number;
    bParticipantPenalties: number;
    aParticipantAdvantages: number;
    bParticipantAdvantages: number;
    aParticipantPoints: number;
    bParticipantPoints: number;
    countdown: number;
    isStarted: boolean;
    isPaused: boolean;
    isCompleted: boolean;

    matchModel: MatchModel;
}